README.md

```diff
@@ -170,3 +170,110 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+#Deployment to Fly.io
+
+
+## Install FlyCtl
+
+To work with the Fly platform, you first need to install Flyctl, 
+a command line interface that allows tou to do everything from creating an account to deploy to Fly.
+
+  - To install it on Linux
+
+    ```  
+      curl  -L https:fly.io/install.sh | sh`
+    ```
+
+- To install it on windows
+    ```
+      iwr https://fly.io/install.ps1 -useb
+    ```
+    
+    Visit `https://fly.io/` if you need installation guide
+  
+
+- Next authenticate with your fly.io account:
+    ```
+     fly auth login`
+    ```
+
+
+- To make sure everything is working well:
+    ```
+     fly apps list
+    ```
+  The output of the above command will be empty table since you have no apps launched yet
+
+
+## Configure the Project
+
+  Environment Variables
+
+  We shouldn't store secrets in source code, so utilizing environmental variables is needed.
+
+  Generate SECRET_KEY (if change needed)
+  - Run the following from the root of your project
+    ```
+    python3 generate_key.py
```

Comment:
Uhm

Reply:
I am going to remove this one, since i generated the SERVER_KEY and put it in .env it's not required to generate neccassarily and providing script to update server key has security problem since it's project identifier.